appengine/genetics/style.css

```diff
@@ -0,0 +1,171 @@
+#blockly {
+    position: fixed;
+    bottom: 10px;
+}
+
+#editor {
+    font-size: 16pt;
+    position: fixed !important;
```

Comment:
As I understand, this is overwriting the CSS for the ACE editor and the second important is overwriting closure CSS so I think it is needed.